addons/website_sale/models/website.py

```diff
@@ -564,32 +564,10 @@ def _display_partner_b2b_fields(self):
 
         return self.is_view_active('website_sale.address_b2b')
 
-    def _get_checkout_steps(self, current_step=None):
-        """ Return an ordered list of steps according to the current template rendered.
-
-        If `current_step` is provided, returns only the corresponding step.
-
-        Note: self.ensure_one()
-
-        :param str current_step: The xmlid of the current step, defaults to None.
-        :rtype: list
-        :return: A list with the following structure:
-            [
-                [xmlid],
-                {
-                    'name': str,
-                    'current_href': str,
-                    'main_button': str,
-                    'main_button_href': str,
-                    'back_button': str,
-                    'back_button_href': str
-                }
-            ]
-        """
+    def _get_checkout_steps_list(self):
```

Comment:
Please add a complete docstring
```suggestion
    def _get_checkout_step_list(self):
        """ Return an ordered list of steps according to the current template rendered.
      
        :rtype: list
        :return: A list with the following structure:
            [
                [xmlid],
                {
                    'name': str,
                    'current_href': str,
                    'main_button': str,
                    'main_button_href': str,
                    'back_button': str,
                    'back_button_href': str
                }
            ]
        """
```